docs/changelog.md

```diff
@@ -1,6 +1,20 @@
 # TBD
+
 ### Features
 * Port over the relevant bits of documentation from `kurtosis-libs`
+* Added new method `GetServiceInfo` in `api_container_service.proto` file
+* Added new method `GetServiceContext` in `NetworkContext` which returns relevant information about the service
+
+### Changes
+* Removed the services map from `NetworkContext` which converts it in stateless network representation
+
+### Breaking Changes
+* Removed `Service` interface
+* Removed `GetService` from `NetworkContext` users can use `GetServiceContext` to get relevant service's information
+* Removed `AvailabilityChecker` class in the returned values of `AddService` and `AddServiceToPartition`
```

Comment:
would recommend adding "; users should either call the service directly to check availability or use the `NetworkContext.WaitForAvailability` method"